### PR TITLE
Add Maven 4.0.0-alpha-8

### PR DESCRIPTION
**Pull request by aalmiray**

### CODE REVIEW

**Review by helpermethod — DISMISSED** [review]
LGTM!

**Review by olamy — CHANGES_REQUESTED** [review]
Please use maven central url

**Comment by aalmiray** [comment]
@helpermethod please hold off the merge til I update the URL, thanks.

**Comment by aalmiray** [comment]
I'm sorry it took so long. The URL has been updated.

cc: @helpermethod

File: src/main/scala/io/sdkman/changelogs/MavenMigration.scala [hunk 1]
```diff
@@ -6,16 +6,15 @@ import com.mongodb.client.MongoDatabase
...
     author = "aalmiray"
   )
-  def migration034(implicit db: MongoDatabase): Unit =
+  def migration035(implicit db: MongoDatabase): Unit =
     Version(
       "maven",
-      "3.9.5",
-      "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.5/apache-maven-3.9.5-bin.zip"
+      "4.0.0-alpha-8",
+      "https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/4.0.0-alpha-8/apache-maven-4.0.0-alpha-8-bin.zipp"
```
Comment (helpermethod) [inline src/main/scala/io/sdkman/changelogs/MavenMigration.scala:17]: That should be `.zip` not `.zipp` 😄

Comment (aalmiray) [reply src/main/scala/io/sdkman/changelogs/MavenMigration.scala]: Fixed 😅

**Review by helpermethod — APPROVED** [review]
LGTM!

**Pull request merged**